frontend/src/pages/Projects.tsx

```diff
@@ -1,7 +1,61 @@
+import { useEffect, useState } from "react"
+import { ProjectDataType } from "../lib/types";
+import FontAwesomeIconWrapper from "../lib/FontAwesomeIconWrapper";
+import { getFilteredIcons } from "../lib/utils";
+import Card from "../components/Card";
+import { level } from "../components/data";
+
+
 export default function Projects() {
+    const [projectData, setProjectData] = useState<ProjectDataType | null>(null);
+
+    useEffect(()=>{
+        document.title = "OWASP Projects"
+        const fetchApiData = async () => {
+            try {
+                const response = await fetch(`${import.meta.env.VITE_BACKEND_BASE_URL}/api/v1/owasp/search/project?q=`)
```

Comment:
Could you use `process.env.API_URL` instead?

This will require adding env_file for the frontend image in docker-compose file

Reply:
In the Vite env is imported as `import.meta.env` as `process.env` does not work and env name must start with `VITE_` prefix
https://vite.dev/guide/env-and-mode#env-files